packages/eslint-plugin/tools/generate-configs.ts

```diff
@@ -9,7 +9,10 @@ function addAutoGeneratedComment(code: string): string {
   return [
     '// THIS CODE WAS AUTOMATICALLY GENERATED',
     '// DO NOT EDIT THIS CODE BY HAND',
-    '// YOU CAN REGENERATE IT USING yarn generate:configs',
+    '// SEE https://typescript-eslint.io/docs/linting/configs',
+    '//',
+    '// For developers working in the typescript-eslint monorepo:',
+    '// You can regenerate it using `yarn generate:configs`',
     '',
     code,
   ].join('\n');
```

Comment:
By the way, I get this lovely error from `chalk` when I try to `yarn generate:configs`:

```plaintext
$ ../../node_modules/.bin/ts-node --files --transpile-only tools/generate-configs.ts
/home/josh/repos/typescript-eslint/packages/eslint-plugin/tools/generate-configs.ts:6
import rules from '../src/rules';
                                ^
Error [ERR_REQUIRE_ESM]: require() of ES Module /home/josh/repos/typescript-eslint/packages/eslint-plugin/node_modules/chalk/source/index.js from /home/josh/repos/typescript-eslint/packages/eslint-plugin/tools/generate-configs.ts not supported.
Instead change the require of index.js in /home/josh/repos/typescript-eslint/packages/eslint-plugin/tools/generate-configs.ts to a dynamic import() which is available in all CommonJS modules.
    at Object.<anonymous> (/home/josh/repos/typescript-eslint/packages/eslint-plugin/tools/generate-configs.ts:6:33)
    at Module.m._compile (/home/josh/repos/typescript-eslint/node_modules/ts-node/dist/index.js:791:29)
    at Object.require.extensions.<computed> [as .ts] (/home/josh/repos/typescript-eslint/node_modules/ts-node/dist/index.js:793:16)
    at phase4 (/home/josh/repos/typescript-eslint/node_modules/ts-node/dist/bin.js:407:16)
    at bootstrap (/home/josh/repos/typescript-eslint/node_modules/ts-node/dist/bin.js:49:12)
    at main (/home/josh/repos/typescript-eslint/node_modules/ts-node/dist/bin.js:32:12)
    at Object.<anonymous> (/home/josh/repos/typescript-eslint/node_modules/ts-node/dist/bin.js:519:5) {
  code: 'ERR_REQUIRE_ESM'
}
```

I resolved it locally by commenting out all the logging. I'm down to just remove the nice pretty printing for now.

Reply:
#5925